### PR TITLE
Add consumer_timeout to config schema

### PR DESCRIPTION
**Pull request by kjnilsson**
https://github.com/rabbitmq/rabbitmq-server/blob/690062880407112e5c373c52f55e502d406cff82/src/rabbit_channel.erl#L506

### CODE REVIEW

**Comment by lukebakken** [comment]
I'll add a test ... one sec

File: priv/schema/rabbit.schema [hunk 1]
```diff
@@ -1068,6 +1068,14 @@ end}.
 {mapping, "vhost_restart_strategy", "rabbit.vhost_restart_strategy",
     [{datatype, {enum, [stop_node, continue, transient, persistent]}}]}.
 
+%% Approximate maximum time a consumer can spend processing a message before
+%% the channel is terminated, in milliseconds.
+%% {consumer_timeout, 10000},
+
+{mapping, "consumer_timeout", "rabbit.consumer_timeout", [
+    {datatype, integer}
```
Comment (michaelklishin) [inline priv/schema/rabbit.schema]: Should we add a "positive integer" validator?

Comment (kjnilsson) [reply priv/schema/rabbit.schema]: If that is an option then yeah.

Comment (lukebakken) [reply priv/schema/rabbit.schema]: OK I'll get that too

**Comment by michaelklishin** [comment]
Backported to `v3.8.x`.

**Pull request merged**